src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/SnackBar.gtk.cs

```diff
@@ -13,7 +14,7 @@ class SnackBar
 	{
 		Timer? snackBarTimer;
 
-		public void Show(Page page, SnackBarOptions arguments)
+		public Task Show(Page page, SnackBarOptions arguments)
```

Comment:
can we make this a `ValueTask`?

Reply:
Return type should be the same for all versions of the method, so it should be `Task` everywhere or `ValueTask` everywhere (including android version).

Should I change it to `ValueTask` everywhere?

Reply:
I believe so, the places that we just return the  `Task.Completed` will not be affected by Task allocation

Reply:
I was wrong. We can use `ValueTask` on some platforms, and `Task` on others because of the way `Show` method is compiled. I thought is was interface, or partial method, or something.
 
 Replaced `Task` with `ValueTask` everywhere except in android implementation.

Reply:
@maxkoshevoi you can use a ValueTask on android as well, that way will be easier to maintain and the compiler will take care of it.